### ISSUE TITLE
Recognise Fortran 2003/2008 file extensions

### ISSUE THREAD

**Issue by hyst329** [Glavin001/atom-beautify#1703]
Related to #1036, but that time it's about Fortran 2003/2008. The Fortran beautifier currently doesn't work with `.f03/.f08` files (as a workaround, changing extension to `.f95` helps).

I suggest adding the "f03", "F03", "f08" and "F08" extensions into https://github.com/Glavin001/atom-beautify/blob/master/src/languages/fortran.coffee
```coffeescript
  extensions: [
    "f90"
    "F90"
    "f95"
    "F95"
    "f03"
    "F03"
    "f08"
    "F08"    
]
```

P.S. Sorry I think that the change is too small for a pull request.
P.P.S. Sorry again for not following the guidelines, since it's not a bug, but a proposal, so it's no need to attach the gists etc.

**Comment by Glavin001** [comment]
> P.S. Sorry I think that the change is too small for a pull request.

If you made a Pull Request instead of creating this Issue I could have simply checked Travis CI passed and merged it! 😉 

> P.P.S. Sorry again for not following the guidelines, since it's not a bug, but a proposal, so it's no need to attach the gists etc.

Agreed.

---

Since I receive many Issues proposing improvements -- like this -- I try to encourage users to contribute smaller fixes themselves and give back to the community. Otherwise I would spend all of my time making these tiny, specific changes which anyone could make, instead of spending my time working on major features which provide value to *all* Atom-Beautify users. This way I also provide the community with an opportunity to contribute and leverage our collective abilities for making Atom-Beautify better!

In conclusion: **Pull Requests welcome!** 😄 I trust you understand my position. As long as Travis CI passes, a simple PR like this will be merged quickly.

**Comment by hyst329** [comment]
Created #1708 pull request fixing this issue.

**Comment by Glavin001** [comment]
Published to v0.30.1
